.github/workflows/go.yml

```diff
@@ -3,20 +3,29 @@ on:
   push:
     branches:
       - master
+  pull_request:
```

Comment:
Please remove this (we only want the pipeline to run pushed to master for the time being)

Reply:
as talked, I will remove this before merging it. keeping it for now to test the PR